pkgs/tools/networking/curl/default.nix

```diff
@@ -126,7 +126,7 @@ stdenv.mkDerivation rec {
     # Install completions
     make -C scripts install
   '' + lib.optionalString scpSupport ''
-    sed '/^dependency_libs/s|${libssh2.dev}|${libssh2.out}|' -i "$out"/lib/*.la
+    sed '/^dependency_libs/s|${lib.getDev libssh2}|${lib.getOutput "out" libssh2}|' -i "$out"/lib/*.la
```

Comment:
The getOutput "out" is not necessary here.

Reply:
Is it wrong to have it for some reason or is it fine either way?

Reply:
I think every package must have an out output even if it is empty or things break very deep.

Reply:
How about `lib.getLib`? Seems to capture the meaning and defaults to `out` when there's no separate `lib` outputs.

Reply:
I'm happy to change it, but somebody with the merge rights needs to tell me which variant I need to push.

Reply:
Well, it looks like you want the output that contains the libraries, so `lib.getLib` is most accurate.

> merge rights

:heavy_check_mark:

Reply:
Updated.